anchore_manager/util/logging.py

```diff
@@ -68,7 +73,9 @@ def log_config(config: dict):
         if config['debug']:
             log_level = 'DEBUG'
 
-        logger.configure_logging(log_level, log_to_stdout=True)
+        enable_json_logging = os.getenv('ANCHORE_JSON_LOGGING_ENABLED') == 'true'
```

Comment:
I understand that the defaults will set this to `"false"`, but it would be safer to fallback to an empty string
```suggestion
        enable_json_logging = os.getenv('ANCHORE_JSON_LOGGING_ENABLED', "") == 'true'
```